projects/igniteui-angular/src/lib/tabs/tabs.directives.ts

```diff
@@ -35,29 +35,36 @@ export class IgxRightButtonStyleDirective {
     }
 
     private getRightButtonStyle(): string {
-        const viewPortWidth = this.tabs.viewPort.nativeElement.offsetWidth;
-
-        // We use this hacky way to get the width of the itemsContainer,
-        // because there is inconsistency in IE we cannot use offsetWidth or scrollOffset.
-        const itemsContainerChildrenCount = this.tabs.itemsContainer.nativeElement.children.length;
-        let itemsContainerWidth = 0;
-        if (itemsContainerChildrenCount > 1) {
-            const lastTab = this.tabs.itemsContainer.nativeElement.children[itemsContainerChildrenCount - 2];
-            itemsContainerWidth = lastTab.offsetLeft + lastTab.offsetWidth;
-        }
-        const headerContainerWidth = this.tabs.headerContainer.nativeElement.offsetWidth;
-        const offset = this.tabs.offset;
-        const total = offset + viewPortWidth;
-
-        // Fix for IE 11, a difference is accumulated from the widths calculations.
-        if (itemsContainerWidth - headerContainerWidth <= 1 && offset === 0) {
+        if (this.tabs.hasContentTabs) {
```

Comment:
Make sure we need this after the tabs template is changed.

Reply:
Addressed.